### DISCUSSION TITLE
Photoprism of this build is not import photos in heic format

### DISCUSSION THREAD

**Discussion by OsipovOV** [tteck/Proxmox#1482] [Help Needed]
When I upload a photo from my Sansung S22 in heic format, I get an error - import: file type heic not supported ... When I upload the same file to https://demo.photoprism.app/, it uploads and displays successfully. 
It looks like ImageMagick is needed to support heic, how do I add it to the PhotoPrism container?

**Comment by fkfreak1234** [comment]
Install

apt install -y libheif-examples exiftool

**Reply by tteck** [reply]
Already installs `exiftool` and `libheif1`
https://github.com/tteck/Proxmox/blob/53b24ef7b77f38a77d8ba3ba4b7bd6c458d4f1d5/install/photoprism-install.sh#L20
https://github.com/tteck/Proxmox/blob/53b24ef7b77f38a77d8ba3ba4b7bd6c458d4f1d5/install/photoprism-install.sh#L22

Of course this is a year old thread.

**Reply by fkfreak1234** [reply]
Ok but i hat this problem today and install this solved the problem

edit: he only install  libheif-examples . After removing this the upload is not working
